extensions/vertx-http/deployment/src/main/resources/dev-templates/main.html

```diff
@@ -61,6 +61,12 @@
    {/if}
    {#insert body/}
   </div>
+
+  {#include logmanager}{/include}
```

Comment:
if you include it, it should probably not have `html > body` in there, no? They'll probably get ignored, but still…

Reply:
Ok could be, I was not sure, let me remove it and see.

Reply:
Done